README.md

```diff
@@ -27,7 +27,7 @@ If you are ready for writing tests, check the [DOCUMENT](./DOCUMENT.md) for the
 ## Install
 
 ```
-$ helm plugin install https://github.com/lrills/helm-unittest
+$ helm plugin install https://github.com/AdamKobi/helm-unittest
```

Comment:
Shouldn't it also be `helm-unittest`?

Reply:
no, we want to install the plugin from this repo